pytorch_lightning/trainer/connectors/fault_tolerant_connector.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import signal
+
+from pytorch_lightning.utilities.imports import _fault_tolerant_training
+
+log = logging.getLogger(__name__)
+
+
+class FaultTolerantConnector:
```

Comment:
what if we rethink our signal handling together with the remains that are in the SLURM connector? 
also we agreed we don't want any connectors modifying the trainer anymore :)

Reply:
Done ! Changed for SignalConnector and merged it with Slurm.